### ISSUE TITLE
Descenders and underscores are cropped in toasters

### ISSUE THREAD

**Issue by stevethepocket** [ValveSoftware/steam-for-linux#2566]
This is happening in KDE, using the built-in text smoothing system. It might be a part of X Window Server or KDE itself; I don't know. The popups in the bottom right have an issue where the text cuts off the bottom pixel of each line. A good example is one I capped here, of my friend who's currently going by the name "de_stroyed". Here's how it looks in the friends window...
![friend](https://f.cloud.github.com/assets/3085348/620586/3d0174ac-cee8-11e2-93b6-979484e096b5.png)
...and here's what his toasters look like:
![toast](https://f.cloud.github.com/assets/3085348/620587/444765b4-cee8-11e2-8634-96acf7315614.png)
As you can see, the underscore is completely cut off, and the y is cropped. As are the "y" and "g" in "is now playing". I saw one earlier where a descender in the name of the game was cropped like that, but I didn't cap it.

**Comment by kisak-valve** [comment]
Hello @stevethepocket, are you still experiencing this issue with an up to date system?

**Comment by stevethepocket** [comment]
Yep. One of my friends just started playing a game, and the "g" was still cut off like before.

Just a heads-up, I've heard it through the grapevine that there's a major UI overhaul in the works. I've only seen a little of it, and that doesn't include the new toasts, but your best bet will probably be to hunt down someone who's working on that and get them to put this on their to-test list. No sense fixing it now if the whole thing is just going to get redone soon.
